BlueDBOnDisk/src/main/java/org/bluedb/disk/serialization/validation/ObjectValidation.java

```diff
@@ -144,4 +145,11 @@ private static boolean isNotEmptyCollection(Object obj) {
 		}
 		return true;
 	}
+
+	private static boolean isByteArray(Field field) {
+		if(field.getType().equals(byte[].class) || field.getType().equals(Byte[].class)) {

```

Comment:
I'm fine with leaving it if you'd like, but could this method be simplified to:
 `return field.getType().equals(byte[].class) || field.getType().equals(Byte[].class);`

Reply:
fixed